master/static/optional_migrations/README.md

```diff
@@ -0,0 +1,16 @@
+# Optional Database Migrations
+
+This directory contains database migrations that are not strictly required but may be desired for 
+some additional functionality. For example, some data migrations for historical data can be 
+time-consuming and not necessary for all use cases. 
+
+Unlike the migrations in `master/static/migrations/`, these migrations will not be run 
+automatically by the master on startup. 
+
+These migrations can be run manually on the Determined database:
+
+```bash
+psql -d determined --single-transaction -f /path/to/migration
+```
+
```

Comment:
```suggestion
This directory contains optional database migrations. While they may be needed to view historical data for some product features, they are time-consuming and not necessary for all use cases.  Unlike the migrations in `master/static/migrations/`, these migrations will not run automatically on cluster startup. Instead, they can be run manually with the following command:
```bash
psql -d $DET_DB_NAME --single-transaction -f $PATH_TO_MIGRATION
```


had fun trying to rewrite this to be more clear. more of an exercise for me lol, lmk what you think.

Reply:
also the ``` destroys github's ability to understand the edit comment..

Reply:
sounds much better than what i had lol, thanks.